source/usage-examples/findOne.txt

```diff
@@ -0,0 +1,51 @@
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+You can retrieve a single document in a collection by chaining together
+the ``find()`` and ``first()`` methods on a ``MongoCollection`` object.
+You can pass a query filter to the ``find()`` method to query for and
+return documents that match the filter in the collection. If you do not
+include a filter, MongoDB returns all the documents in the collection. The
+``first()`` method returns the first document from the matches.
+
+For more information on querying MongoDB with the Java driver, see our
+guide (TODO: link) on querying documents.
+
+You can also chain other methods to the ``find()`` method
+such as ``sort()`` (TODO: link) which organizes the matched documents in a
+specified order, and ``projection`` (TODO: link) which configures the
+included fields in the return document.
+
+The ``find()`` returns an instance of an iterator. Iterators are helpers that
```

Comment:
This sentence reads a bit awkwardly to me. I'd suggest either changing this to "The ``find()`` method" or just trimming "The" from the beginning of the sentence.